README.md

```diff
@@ -41,10 +48,73 @@ This will ensure you install the exact packages that we've tested.
 > - At present we've tested the pipeline only on Centos 7 and Rocky 9 Linux and using Python 3.7.
 > - The provided environment does not include additional packages required for specific GPU support, e.g. CUDA.
 
+### Build and use Apptainer/Singularity container
+
+To build an Apptainer/Singularity container, you can use the provided `Cellos.def` file (either clone the repository or wget/curl the `.def` file):
+
+```bash
+apptainer build Cellos.sif Cellos.def
+```
+
+> [!NOTE]
+> As a convenience, we also provide a `Dockerfile`, however this has not been extensively tested and is not supported in our computing environment, so we cannot offer any support or universal advice on using it.
+> 
+> To build the Docker container, you can use the provided `Dockerfile` after cloning the repository:
+> 
+> ```bash
+> docker build -t Cellos .
+> ```
+
+
+#### Building the Apptainer/Singularity container on Sumner2
+
+> [!TIP]
+> At JAX, the easiest way to build containers from the definitions in this repository is to use the `build` partition on Sumner2.
+> For more details regarding accessing and using this JAX-specific resource, please see [the instructions in SharePoint](https://jacksonlaboratory.sharepoint.com/sites/ResearchIT/SitePages/JAX-HPC-Pro-Tip.aspx).
+
+You can access it from a login node by using:  
+```bash
+sinteractive -p build -q build
+```
+
+> [!TIP]
+>  `singularity` and `apptainer` can be used interchangeably in all of these commands.
+
+Load the needed singularity/apptainer module:
+```
+module load singularity
+```
+
+To build the container either clone the repository:
+```
+git clone https://github.com/TheJacksonLaboratory/Cellos.git
+```
+or download the container definition:
+```
+wget https://github.com/TheJacksonLaboratory/Cellos/raw/refs/heads/master/Cellos.def
```

Comment:
This URL will not work till this is merged. I think it should be correct, but if it's wrong we will fix it!